src/tests/ftest/rebuild/cascading_failures.yaml

```diff
@@ -11,8 +11,12 @@ hosts:
 timeout: 360
 server_config:
   name: daos_server
-  servers:
-    targets: 8
+  servers: !mux
+#skipForTicket DAOS-2799
+#    8_targets:
+#      targets: 8
+    1_target:
+      targets: 1
```

Comment:
We should not be skipping variants by commenting them out.  Skipping a test variant should be implemented through the new apricot.Test.CANCEL_FOR_TICKET list of lists.  In this case it would be implemented here in `rebuild/cascading_failures.py`:
```
class CascadingFailures(RebuildTestBase):
    # pylint: disable=too-many-ancestors
    """Test cascading failures during rebuild.
    :avocado: recursive
    """

    CANCEL_FOR_TICKET = [["DAOS-2799", "targets", 8]]
```

The issue with adding `servers: !mux` and including `targets: 1` is this will increase the test count from 3 (`test_simultaneous_failure`, `test_sequential_failures`, `test_cascading_failures`) to 6 (with each of the 3 test methods now running two variants each).  It's unclear if this is intended or not.  If it is not intended then a new set of files (*.py & *.yaml) should be created to run the specific test method with it's two variants and that method should be removed from the original file.

Having said all of that, all three methods in `rebuild/cascading_failures.py` are currently skipped with the `@skipForTicket` decorator, so one could argue that these changes to the yaml file are not even needed and it should move back to just specifying 8 targets.

Reply:
Please see the new commit.